wicket-extensions/src/main/java/org/apache/wicket/extensions/markup/html/repeater/data/table/NavigationToolbar.java

```diff
@@ -73,16 +68,16 @@ protected PagingNavigator newPagingNavigator(final String navigatorId,
 	}
 
 	/**
-	 * Factory method used to create the navigator label that will be used by the datatable
+	 * Factory method used to create the navigator label.
 	 * 
 	 * @param navigatorId
 	 *            component id navigator label should be created with
 	 * @param table
-	 *            dataview used by datatable
+	 *            DataTable used by datatable
 	 * @return navigator label that will be used to navigate the data table
 	 * 
 	 */
-	protected WebComponent newNavigatorLabel(final String navigatorId, final DataTable<?, ?> table)
+	protected Component newNavigatorLabel(final String navigatorId, final DataTable<?, ?> table)
```

Comment:
Please do the same for `org.apache.wicket.extensions.ajax.markup.html.AjaxEditableChoiceLabel#newLabel()`

Reply:
Done